src/core/Akka.Cluster.Tests.MultiNode/MultiNodeClusterSpec.cs

```diff
@@ -31,7 +31,7 @@ public abstract class MultiNodeClusterSpec : MultiNodeSpec
         public static Config ClusterConfigWithFailureDetectorPuppet()
         {
             return ConfigurationFactory.ParseString(
-                @"akka.cluster.failure-detector.implementation-class = ""Akka.MultiNodeTests.FailureDetectorPuppet, Akka.MultiNodeTests""")
+                string.Format(@"akka.cluster.failure-detector.implementation-class = ""{0}""", typeof(FailureDetectorPuppet).AssemblyQualifiedName))
```

Comment:
changed the FQN we pass to HOCON for the `FailureDetectorPuppet` to be something we load at runtime based on reflection.